numpy/lib/twodim_base.py

```diff
@@ -136,11 +205,11 @@ def flipud(m):
     return m[::-1, ...]
 
 
-def rot90(m, k=1):
+def rot90(m, k=1, axes=(0,1)):
     """
     Rotate an array by 90 degrees in the counter-clockwise direction.
```

Comment:
It might be helpful to describe it as rotation towards the first axis from
the second axis.
On Sat, Feb 27, 2016 at 11:05 AM Denis Alevi <notifications@github.com>
wrote:

> In numpy/lib/twodim_base.py
> <https://github.com/numpy/numpy/pull/7347#discussion_r54335090>:
>
> > @@ -136,11 +205,11 @@ def flipud(m):
> >      return m[::-1, ...]
> >
> >
> > -def rot90(m, k=1):
> > +def rot90(m, k=1, axes=(0,1)):
> >      """
> >      Rotate an array by 90 degrees in the counter-clockwise direction.
>
> I change the doc to make it clearer and added tests for different axes
> order here d2e85e0
> <https://github.com/erensezener/numpy/commit/d2e85e0a30d5524e3ee42290be2314bc3117d78e>
>
> I would have said counter-clockwise rotation is defined in a 2-D plane
> independent of what is rotated in that plane. But I don't know, what that
> means for a N-D array rotating in that plane. So, if you would say I should
> discard the term counter-clockwise entirely, I can do so.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/7347/files#r54335090>.
>